### PR TITLE
feat(windows-sign): add unstable ssl.com windows signing

### PR DESCRIPTION
**Pull request by Gankra**

### CODE REVIEW

**Comment by Gankra** [comment]
main outstanding question is names of the SECRETS which I made up and don't match ssl.com's defaults. They use ES_USERNAME, ES_PASSWORD, CREDENTIAL_ID, and ES_TOTP_SECRET ("es" is "esign").

File: cargo-dist/templates/ci/github_ci.yml.j2 [hunk 1]
```diff
@@ -177,20 +177,82 @@ jobs:
...
+          mkdir -p "$SIGN_DIR_OUT"
+          for file in target/distrib/*.{msi,ps1}; do
+            [[ -e $file ]] && mv "$file" "$SIGN_DIR_IN" && echo "signing $file";
+          done
+      # Sign the files
+      - name: Sign Artifacts with CodeSignTool
+        uses: sslcom/esigner-codesign@develop
+        with:
+          command: batch_sign
+          username: ${{ secrets.SSL_CODESIGN_USER }}
```
Comment (mistydemeo) [inline cargo-dist/templates/ci/github_ci.yml.j2:215]: I think using an `SSLCOM` prefix would make this a little clearer to me.

**Review by mistydemeo — APPROVED** [review]

File: cargo-dist/templates/ci/github_ci.yml.j2 [hunk 2]
```diff
@@ -177,20 +177,82 @@ jobs:
...
+
+  # Sign Windows artifacts with ssl.com
+  sign-windows-artifacts:
+    needs:
+      - plan
+      - upload-local-artifacts
+    {{%- if global_task %}}
+      - upload-global-artifacts
+    {{%- endif %}}
+    runs-on: "ubuntu-20.04"
```
[the remark this answers is not in the document]
Comment (Gankra) [reply cargo-dist/templates/ci/github_ci.yml.j2:190]: yeah it's pure java, running it on windows is actually harder (they ship the whole jvm x3)

File: cargo-dist/templates/ci/github_ci.yml.j2 (hunk 1 above)
Comment (Gankra) [reply cargo-dist/templates/ci/github_ci.yml.j2]: I changed everything to SSLDOTCOM/ssldotcom prefixes (matches how we treat axo.dev) and made these match a bit more with the action's defaults.

**Pull request merged**